example/pvc-test.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: example-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: test-csi-mock 
```

Comment:
missing a storageclass yaml?

Reply:
The storageclass is included in deploy/csi-mock-driver-deployment.yaml